client/extensions/woocommerce/state/data-layer/ui/products/test/index.js

```diff
@@ -2,136 +2,222 @@
  * External dependencies
  */
 import { expect } from 'chai';
+import { spy } from 'sinon';
 
 /**
  * Internal dependencies
  */
-import { makeProductActionList, } from '../';
+import { handleProductCategoryEdit, makeProductActionList, } from '../';
 import { actionListStepSuccess, actionListStepFailure } from 'woocommerce/state/action-list/actions';
 import { createProduct } from 'woocommerce/state/sites/products/actions';
+import { editProductRemoveCategory } from 'woocommerce/state/ui/products/actions';
+import { editProductCategory } from 'woocommerce/state/ui/product-categories/actions';
+
+describe( 'handlers', () => {
+	describe( '#handleProductCategoryEdit', () => {
+		const existingCategory = { id: 101, name: 'Existing Category' };
+		const existingCategoryCreate = { id: { placeholder: 'productCategory_1' }, name: 'New Category' };
```

Comment:
I changed it to `newCategory1` and the other new category to `newCategory2`.
I also change the product to `newProduct`.